src/app/shared/services/transfer-post.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Post } from "../interfaces";
+
+export class TransferPostService {
+  private _post!: Post
+
+  set post(selectedPost: Post) {
+    this._post = selectedPost;
+  }
+
+  get post() : Post {
+    return this._post;
+  }
+}
```

Comment:
why do you need this service?

Reply:
I used it as a storage for new post creation. It transfers new Post to a component which lists all of them.